evmd/go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	cosmossdk.io/x/upgrade v0.2.0
 	github.com/cometbft/cometbft v0.38.17
 	github.com/cosmos/cosmos-db v1.1.3
-	github.com/cosmos/cosmos-sdk v0.53.3
+	github.com/cosmos/cosmos-sdk v0.53.4-0.20250725182846-908df9d4e2e0
```

Comment:
Do we need to use this hash or can we use an actual release tag?

Reply:
can use the release tag: https://github.com/cosmos/evm/pull/387/commits/99b8e772a9353715ae4b4cf085501a78866dc1a5